script.js

```diff
@@ -0,0 +1,7 @@
+function myFunction()
```

Comment:
Always a good idea to name functions what they do. Something like `sum()` or `calculate()`. As projects get bigger, having clear consistent naming is the difference between sanity and insanity 😄